README.md

```diff
@@ -1,3 +1,57 @@
-# Pathways [![Build Status](https://travis-ci.org/mcode/pathways.svg?branch=master)](https://travis-ci.org/mcode/pathways)
+![camino-logo-light](https://user-images.githubusercontent.com/13512036/73880187-ec17d580-482b-11ea-9b4f-e5f15a492fad.png)
+# Camino: mCODE™-based Oncology Clinical Pathways Prototype
+[![Build Status](https://travis-ci.org/mcode/pathways.svg?branch=master)](https://travis-ci.org/mcode/pathways)
 
-mCODE-based Oncology Clinical Pathways Prototype
+
+## mCODE: Minimal Common Oncology Data Elements
+mCODE is an initiative intended to assemble a core set of structured data elements for oncology electronic health records (EHRs). mCODE is a step towards capturing research-quality data from the treatment of all cancer patients. This would enable the treatment of every cancer patient to contribute to comparative effectiveness analysis of cancer treatments by allowing for easier methods of data exchange between health systems. mCODE has been created and is being supported by The American Society of Clinical Oncology (ASCO®) in collaboration with the MITRE Corporation. Read more at https://mcodeinitiative.org/ .
+
+## Camino
```

Comment:
I feel like the Camino section should be above mCODE because that's what the repo is for.  Whatever works though.